src/source/PeerConnection/SessionDescription.c

```diff
@@ -160,10 +160,13 @@ STATUS setPayloadTypesFromOffer(PHashTable codecTable, PHashTable rtxTable, PSes
     UINT16 aptFmtVal;
     BOOL supportCodec;
     UINT32 tokenLen, i, aptFmtpValCount;
+    PCHAR fmtp;
+    INT32 fmtpScore, bestFmtpScore;
```

Comment:
Always use unsigned integers unless you really need signed. In this case, there is no need for signed so use UINT32

Reply:
Ok, will combine with other feedback and switch to DOUBLE, so as to allow scoring between [0, 1]

Reply:
DOUBLE sounds good as well. Though, it would be funny defining the range. Using UINT32 would allow to simply add the "features" and return the value as the score, allowing later to expand